server/models/Profile.js

```diff
@@ -0,0 +1,43 @@
+const mongoose = require("mongoose");
+
+const userProfileSchema = new mongoose.Schema({
+  firstName: {
+    type: String,
+    required: true,
+  },
+  lateName: {
+    type: String,
+    required: true,
+  },
+  gender: {
+    type: String,
+    required: true,
+  },
+  birthday: {
+    type: Date,
+    required: true,
+  },
+  email: {
+    type: String,
+    required: true,
+  },
+  phoneNumber: {
+    type: String,
+    required: true
+  },
+  address: {
+    type: String,
+    required: true
+  },
+  description: {
+    type: String,
+  },
+  user: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'User'
```

Comment:
```suggestion
    ref: 'User',
    required: true
```
Let's make this a required attribute :)